docs/contracts/NFT Contract/overview.md

```diff
@@ -134,3 +134,16 @@ The **setconrecv** action allows the _Asset Manager_ to set the token factory co
 
 -   [setconrecv - set conditionless receivers](./NFT%20Actions/setconrecv.md)
 -   [factory.b](./nft-tables.md#factory-b)
+
+## Adding factory purchase options
+
+The _Asset Manager_ can add the factory first-hand purchase requirements
```

Comment:
In another file you use **Asset Manager**. You must pick a style.

Reply:
It's the problem with this page as a whole, it uses a different style compared to other pages so I made this section follow the style of the rest of the document